types/src/contracts.rs

```diff
@@ -1675,3 +1675,27 @@ mod tests {
         assert_eq!(contract_hash, decoded)
     }
 }
+
+#[cfg(test)]
+mod prop_tests {
+    use proptest::prelude::*;
+
+    use crate::{bytesrepr, gens};
+
+    proptest! {
+        // #![proptest_config(ProptestConfig {
```

Comment:
Do we need this?

Reply:
It's just very handy - easy to re-introduce to code and navigate into relevant `proptest` code.